src/peergos/server/Start.java

```diff
@@ -109,7 +109,9 @@ public static void startPeergos(Args a) {
 
             CoreNode core = HTTPCoreNode.getInstance(coreAddress);
             MutablePointers mutable = HttpMutablePointers.getInstance(coreAddress);
-            MutablePointers pinner = new PinningMutablePointers(mutable, dht);
+            String blacklistPath = "blacklist.txt";
```

Comment:
Can this be parameterized?

Reply:
I think it's fine to require the blacklist.txt name no? They can always symlink if they really want it elsewhere.